content/2fa-services/index.md

```diff
@@ -0,0 +1,132 @@
+cover: ""
+category: "programming"
+author: "Darko Bozhinovski"
+---
+
+Let's talk about two-factor authentication (2FA) - what it is, why it's important, and why you probably need to use it. Sure, it's a sometimes annoying extra step your users *may* complain about. Still, it's a small price to pay for the security it provides - not to mention it's exactly what stands between their accounts and the proverbial "script kiddies" with too much time on their hands, professional hackers, and other malicious actors.
+
+The problem, though, is that, as with many things in software development, we're spoiled for choice. And the experience of picking one can be as fun as debugging regex. I don't imagine many of us enjoy comparing pages upon pages of features, pricing, and other details. You've got costs that scale faster than that infinite loop AWS lambda you forgot about, integration headaches that make the OAuth RFC look like a light weekend read, and user experiences ranging from 🤷 to "Why do you hate me?"
+
+Still with me? Cool. Let's fix that by having a look at some options out there.
+
+## 1. SuperTokens: Simplified 2FA for Developers
+
+There's a gradient between "build auth yourself" (and potentially introduce various footguns), and "go bankrupt and use BigAuth". SuperTokens lies in the middle - you can self-host it, put it up on a cloud provider of your choice, go managed (without us doing the cost-magically-increases rug pull), mix and match, and/or migrate as the need arises. 
+
+Why it might work for you:
+
+- Pre-built UI that doesn't look like it's from cca 2010. It's also fully customizable.
+- Multi-tenant support that works as advertised.
+- TOTP support, for when you absolutely and positively must use Google Authenticator.
+- No charges for dev licenses - which is an article unto itself considering the offering out there, and best left for another time.
+- Easily integrates with a plethora of backend and frontend frameworks out there.
+
+More importantly to 2FA, SuperTokens comes with OTP via SMS and email. The best part? You won’t need a PhD in cryptography to implement it. Plus, the pricing won’t make your CFO faint - it's $0.01 / MAU  for the managed version (with a minimum charge of $100).
+
+## 2. Duo Security
+
+Suppose enterprise is more your jam, and you’re looking for something that scales well and comes with enterprise features without the enterprise complexity. In that case, Duo Security (now a part of Cisco) might be the answer you're looking for. 
+
+Why it might work for you:
+
+- Clean, modern UI.
+- Strong focus on large organizations with an enteprise-grade 2FA.
+- Push notifications, SMS, and biometric verification
+- Free tier (up to 10 users); Pricing starts at $3 per user per month on the Essentials plan.
+
+While it makes a lot of sense to use Duo for enterprise-grade 2FA, it might be a bit too much for smaller organizations. 
```

Comment:
Could add a line to substantiate this a bit more. Is it only the 3$ pricing, or is there some implementation complexity? Would smaller organizations not use some of the "enterprise" features, hence the value proposition is not there for them?